site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs

```diff
@@ -4,12 +4,35 @@
 	"language": "en",
 	"description": "v4.0 changelog - Version history - Web Experience Toolkit (WET)",
 	"altLangPrefix": "v4.0-changelog",
-	"dateModified": "2017-05-17"
+	"dateModified": "2017-08-15"
 }
 ---
 <p>See <a href="../version-info-en.html">Versioning</a> for the meaning of each version number.</p>
 <span class="wb-prettify all-pre"></span>
 <section>
+	<h2>v4.0.26</h2>
+	<ul>
+		<li><strong>Release date:</strong> <time>2017-08-15</time></li>
+		<li><a href="v4.0.26-en.html">Release notes<span class="wb-inv"> - v4.0.26</span></a></li>
+		<li><a href="../dwnld-en.html#v4026">Downloads<span class="wb-inv"> - v4.0.26</span></a></li>
+	</ul>
+
+	<section>
+		<h3>What's New</h3>
+		<ul>
+			<li>Used pre-built docker hub image for building</li>
+			<li>Switched to a docker CI build</li>
```

Comment:
Don't need to include these two in the what's new because it doesn't affect the output